iOS/DuckDuckGo/HomeMessageViewModel.swift

```diff
@@ -164,6 +165,12 @@ struct HomeMessageViewModel {
             }
         }
     }
+    
+    /// If `last_search_state` is present, refresh before opening URL
+    private func refreshLastSearchState(in urlString: String) -> String {
+        let lastSearchDate = AutofillUsageStore().searchDauDate
+        return DefaultRemoteMessagingSurveyURLBuilder.refreshLastSearchState(in: urlString, lastSearchDate: lastSearchDate)
+    }
```

Comment:
Looks good to me, I can work on a change to make this happen automatically next week, so that params are always up to date.